util__deploy_to_maven_central.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+#------------------------------------------------------------
+VERSION=`mvn help:evaluate -Dexpression=project.version | grep -v -e "^\["`
```

Comment:
Can you use `$(command)` instead of `` `command` ``?

Reply:
Done.